ic-utils/Cargo.toml

```diff
@@ -16,10 +16,9 @@ include = ["src", "Cargo.toml", "../LICENSE", "README.md"]
 
 [dependencies]
 async-trait = "0.1.40"
-candid = "0.7"
+candid = "0.7.1"
```

Comment:
I believe excluding the minor version here is intended, since this is a library.

Reply:
Same as above.